src/js/main.js

```diff
@@ -5,8 +5,20 @@ var analytics = undefined;
 
 $(document).ready(function () {
     $.getJSON('version.json', function(data) {
+        CONFIGURATOR.version = data.version;
         CONFIGURATOR.gitChangesetId = data.gitChangesetId;
 
+        // Version in the ChromeApp's manifest takes precedence.
+        if(chrome.runtime && chrome.runtime.getManifest) {
+            var manifest = chrome.runtime.getManifest();
+            CONFIGURATOR.version = manifest.version;
+            // manifest.json for ChromeApp can't have a version
+            // with a prerelease tag eg 10.0.0-RC4
+            // Work around is to specify the prerelease version in version_name
+            if (manifest.version_name) {
+                CONFIGURATOR.version = manifest.version_name;
+            }
+        }
```

Comment:
@mikeller I like the idea of offering a pre-release build since there is an opt-in option for it. how about this? 

I've added a comment to explain `version_name`, as it wasn't clear when I tore it out